var/spack/repos/builtin/packages/c-blosc/test_forksafe.patch

```diff
@@ -0,0 +1,10 @@
+--- a/tests/test_forksafe.c	2019-07-30 15:01:36.208670230 -0500
++++ b/tests/test_forksafe.c	2019-07-30 15:02:27.475543401 -0500
+@@ -43,5 +43,6 @@
+   int success = 0;
+   int status = 0;
+-  for(float sec = 0; sec < 1; ) {
++  float sec = 0;
++  while (sec < 1) {
```

Comment:
Just curious: what is the error you get without this patch? This seems to be doing what a `for` loop should do, but without keeping `sec` in the `for` loop scope.

Reply:
@alalazo Not a silly question. I should have provided more info. This is related to #12110 where for some reason `test_forksafe` would fail with 'sec undefined'. 
```console
...
tests/test_forksafe.c
            (45): error: identifier "sec" is undefined
     448        for(float sec = 0; sec < 1; ) {
...
```
I posted an issue to the c-blosc project https://github.com/Blosc/c-blosc/issues/271. The person who looked at the issue could not reproduce it but made a patch in the c-blosc repository anyway. I tested with many versions of c-blosc in spack and all had trouble with that file without the patch with icc, but succeed with the patch. I did have to alter the context lines of the patch to get it to work across all versions.